lib/models/config.ts

```diff
@@ -18,13 +18,53 @@ export interface ConfigDocument extends Document {
   value: string
 }
 
+export const registrationMode: Record<string, any> = {
+  [SECURITY_REGISTRATION_MODE_OPEN]: '公開 (だれでも登録可能)',
+  [SECURITY_REGISTRATION_MODE_RESTRICTED]: '制限 (登録完了には管理者の承認が必要)',
+  [SECURITY_REGISTRATION_MODE_CLOSED]: '非公開 (登録には管理者による招待が必要)',
+}
+
+export function isRequiredThirdPartyAuth(config) {
```

Comment:
Just question, why did you move static method to just functions? I have a concern about mixing functions and static methods at the model layer.

Reply:
BTW, I think it's better to add a type.

```
export function isRequiredThirdPartyAuth(config: Config) {
```

Reply:
> why did you move static method to just functions

IMO, we can make Config model's function simpler with separating static model functions and these simple utility function, 
And it could be imported as a single utility function without model instance from anywhere.

Reply:
That makes sense. You meant you wanted to get some information from the config object like calling Config document object method without Config model's static method but you can't because it is a plain JavaScript object, didn't you?
I just had a concern about how to separate static method and plain functions in all models. However, I'm okay that making Config model's static methods for getting some information to be a function instead of making all static methods of all models to be a function as much as possible.

In the future, I think we can make a config object to be an entity class. Then, we don't have to call functions and dependent Config model.